### ISSUE TITLE
[feature request] Possibility to disable pin-code for web configurator

### ISSUE THREAD

**Issue by RC-Thoughts** [unfoldedcircle/bug-reports#194]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

I'd like to see a possibility to access web configurator without the use of pin..

**Comment by drcursor** [comment]
Alternatively, an option to set a static PIN.

**Comment by kennymc-c** [comment]
@drcursor The PIN is already static if you never manually rotate it

**Comment by drcursor** [comment]
But can you set it ?

**Comment by kennymc-c** [comment]
No , but you can e.g. save it to a password manager

**Comment by kennymc-c** [comment]
Instead of disabling the PIN completely you could also have to option mark a known device to not ask for a PIN again on this device.
The session timeout should also be extented as it seems as it's only as long as the browser tab is open.

**Comment by acurcione** [comment]
I’d love for it not to rotate. I just disabled/enabled the configurator cause I thought it had crashed and wasn’t accessible. Instantly reset the pin. The pin should either be able to be set and stored or disabled completely.

**Comment by pkmccarthy** [comment]
@kennymc-c  - maybe I am mistaken, but on a remote reboot, the PIN changes on its own, correct?  so, a Static PIN would be helpful.

**Comment by kennymc-c** [comment]
@pkmccarthy No,the PIN only changes if you manually refresh it or enable/disable the web configurator on the remote.
